src/components/List/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Meta, Story } from '@storybook/react';
+
+import { Divider } from '@components/Divider';
+
+import { List as ListComponent, ListProps } from '../List';
+import { ListItem as ListItemComponent } from '../components';
+import { ListItemText as ListItemTextComponent } from '../components';
```

Comment:
can we import it like that 

import {
  ListItem as ListItemComponent,
  ListItemText as ListItemTextComponent,
} from '../components';?

Reply:
yes we can I actually added `index,ts` later in components forgot to update imports here